scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/InMemoryConnectionStore.java

```diff
@@ -263,4 +265,9 @@ public void sessionEstablished(final Handshaker handshaker, final DTLSSession es
 	public void handshakeCompleted(final InetSocketAddress peer) {
 		// nothing to do
 	}
+
+	@Override
+	public void handshakeFailed(final InetSocketAddress peer, HandshakeException exception) {
+		// nothing to do
```

Comment:
As far as I understood, this is already done before. I didn't want to change the current call logic.